### PR TITLE
[js] HTML extern minor improvements: keywords and docs

### PR DESCRIPTION
**Pull request by haxiomic**
Minor improvements:

- Remove trailing underscore from non-keywords such as `DEFAULT_` or `delete_`
- Add documentation that was missing because of trailing underscore

### Impact
Minimal to no impact on existing code - I've found no references to any of these fields in any of the popular libraries

### CODE REVIEW

File: std/js/html/FormData.hx [hunk 1]
```diff
@@ -43,6 +43,10 @@ extern class FormData
 	**/
 	@:overload( function( name : String, value : Blob, ?filename : String ) : Void {} )
 	function append( name : String, value : String ) : Void;
+	
+	/**
+		Deletes a key/value pair from a `FormData` object.
+	**/
 	@:native("delete")
 	function delete_( name : String ) : Void;
```
Comment (nadako) [inline std/js/html/FormData.hx]: Hm, not related to this particular change, but `delete` is not a keyword in Haxe, so this shouldn't be needed.

File: std/js/html/LinkElement.hx [hunk 2]
```diff
@@ -102,6 +102,10 @@ extern class LinkElement extends Element
 	**/
 	var target : String;
 	var integrity : String;
+	
+	/**
+		Is a `DOMString` representing the type of content being loaded by the HTML link.
+	**/
 	@:native("as")
 	var as_ : String;
```
Comment (nadako) [inline std/js/html/LinkElement.hx]: Same remark as for the `delete` above.

File: std/js/html/SpeechSynthesisVoice.hx [hunk 3]
```diff
@@ -54,6 +54,10 @@ extern class SpeechSynthesisVoice
 		A `Boolean` indicating whether the voice is supplied by a local speech synthesizer service (`true`), or a remote speech synthesizer service (`false`.)
 	**/
 	var localService(default,null) : Bool;
+	
+	/**
+		A `Boolean` indicating whether the voice is the default voice for the current app language (`true`), or not (`false`.)
+	**/
 	@:native("default")
 	var default_(default,null) : Bool;
```
Comment (nadako) [inline std/js/html/SpeechSynthesisVoice.hx:62]: Here too

File: std/js/html/FormData.hx (hunk 1 above)
Comment (haxiomic) [reply std/js/html/FormData.hx]: Good catch @nadako! Not sure where the original keyword list is from but I'll synchronize it with `ast.ml` and update this PR

File: std/js/html/SpeechSynthesisVoice.hx (hunk 3 above)
Comment (haxiomic) [reply std/js/html/SpeechSynthesisVoice.hx:62]: This one is ok – default is a keyword

**Pull request merged**